packages/ui/src/theme/types.ts

```diff
@@ -39,7 +39,7 @@ interface BaseOverride {
  * ```
  */
 interface BreakpointOverride extends BaseOverride {
-  breakpoint: keyof Breakpoints['values'];
+  breakpoint: keyof Breakpoints['values'] | (string & {});
```

Comment:
Forgive me if i do not fully understand the need for this change, but was hoping you could clarify a couple things for me:
1. Doesn't `(string & {})` always enforce just a `string` making the `{}` unnecessary?
<img width="684" alt="image" src="https://user-images.githubusercontent.com/15002885/181596403-6c0dc5cc-f8c9-46a2-9354-6ff24cff4e88.png">

2. Is there a specific reason (that I am missing) for wanting to remove type strictness from `breakpoint`? It seems like allowing any `string` value for `breakpoint` kind of defeats the purpose of type safety?